lib/housekeeping.js

```diff
@@ -136,20 +141,44 @@ class HouseKeeper {
         volumesMonitor.measure('${region}.type:${type}.count', combinedCount);
         totalGbUsage += combinedGbUsage;
         totalCount += combinedCount;
-      });
-    });
-    
+        
+        let transaction = await this.state.beginTransaction();
+        try {
```

Comment:
If I remember correctly, we want to ensure that rows don't "stick around" in this table if they are no longer being updated -- for example, if we stop monitoring a region.  I think the easiest way to accomplish that is simply to delete all rows from the table at the beginning of the transaction.  In that case, you won't need the upsert, either -- an insert will do, since you know the table is otherwise empty.